<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>591</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>对象</string>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>252</red>
        <green>121</green>
        <blue>27</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>类型</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>路径</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>981</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menulingcunwei">
     <property name="title">
      <string>另存为</string>
     </property>
     <addaction name="actionwenbenwenjian"/>
     <addaction name="actioncsv"/>
     <addaction name="actionweitu"/>
    </widget>
    <addaction name="actiondaoru"/>
    <addaction name="actiondaochu"/>
    <addaction name="menulingcunwei"/>
    <addaction name="actioncunchu"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>绘图</string>
    </property>
    <widget class="QMenu" name="menu_IPF">
     <property name="title">
      <string>绘制IPF图</string>
     </property>
     <addaction name="actionIPF_x"/>
     <addaction name="actionIPF_y"/>
     <addaction name="actionIPF_z"/>
    </widget>
    <addaction name="actionhuizhiEular"/>
    <addaction name="menu_IPF"/>
    <addaction name="actionhuizhiBM"/>
    <addaction name="separator"/>
    <addaction name="actiontianjiatuli"/>
    <addaction name="actiontianjiabilichi"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>计算</string>
    </property>
    <addaction name="actionchonggou"/>
    <addaction name="separator"/>
    <addaction name="actionchonggoushezhi"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="actionchongxinxuanzejinxiang"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>显示</string>
    </property>
    <addaction name="actionzhanshitingkaowu"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>框选</string>
    </property>
    <addaction name="actionwanchengkuangxuan"/>
    <addaction name="separator"/>
    <addaction name="actionkuangxuanshezhi"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>可能可以再添加一栏</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
   <addaction name="menu_6"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>WIDTH</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>MODE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>COLOR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>框选</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actiondaoru">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="actiondaochu">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="actionhuizhiEular">
   <property name="text">
    <string>绘制欧拉角图</string>
   </property>
  </action>
  <action name="actionIPF_x">
   <property name="text">
    <string>IPF-x</string>
   </property>
  </action>
  <action name="actionIPF_y">
   <property name="text">
    <string>IPF-y</string>
   </property>
  </action>
  <action name="actionIPF_z">
   <property name="text">
    <string>IPF-z</string>
   </property>
  </action>
  <action name="actionhuizhiBM">
   <property name="text">
    <string>绘制BM图</string>
   </property>
  </action>
  <action name="actionchongxinxuanzejinxiang">
   <property name="text">
    <string>重新选择进向</string>
   </property>
  </action>
  <action name="actiontianjiatuli">
   <property name="text">
    <string>添加图例</string>
   </property>
  </action>
  <action name="actiontulishezhi">
   <property name="text">
    <string>图例设置</string>
   </property>
  </action>
  <action name="actionzhanshitingkaowu">
   <property name="text">
    <string>展示停靠坞</string>
   </property>
  </action>
  <action name="actionwanchengkuangxuan">
   <property name="text">
    <string>完成框选</string>
   </property>
  </action>
  <action name="actionkuangxuanshezhi">
   <property name="text">
    <string>框选设置</string>
   </property>
   <property name="toolTip">
    <string>框选设置</string>
   </property>
  </action>
  <action name="actionsuoxuanziwenjian">
   <property name="text">
    <string>所选子文件</string>
   </property>
  </action>
  <action name="actionwenbenwenjian">
   <property name="text">
    <string>文本文件</string>
   </property>
  </action>
  <action name="actioncsv">
   <property name="text">
    <string>csv文件</string>
   </property>
  </action>
  <action name="actioncunchu">
   <property name="text">
    <string>保存所有文件</string>
   </property>
  </action>
  <action name="actionweitu">
   <property name="text">
    <string>bmp位图</string>
   </property>
  </action>
  <action name="actiontianjiabilichi">
   <property name="text">
    <string>添加比例尺</string>
   </property>
  </action>
  <action name="actionchonggou">
   <property name="text">
    <string>重构</string>
   </property>
  </action>
  <action name="actionchonggoushezhi">
   <property name="text">
    <string>重构设置</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
